src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -0,0 +1,139 @@
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import type {CentralPaneNavigatorParamList} from '@navigation/types';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import WorkspacePageWithSections from '@pages/workspace/WorkspacePageWithSections';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+import ToggleSettingOptionRow from './ToggleSettingsOptionRow';
+import type {OptionType} from './ToggleSettingsOptionRow';
+
+type WorkspaceWorkflowsPageProps = WithPolicyProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.WORKFLOWS>;
+
+function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {isSmallScreenWidth} = useWindowDimensions();
+
+    const items: OptionType[] = [
+        {
+            icon: Illustrations.ReceiptEnvelope,
+            title: translate('workflowsPage.delaySubmissionTitle'),
+            subtitle: translate('workflowsPage.delaySubmissionDescription'),
+            onToggle: (isEnabled: boolean) => {
+                Policy.setWorkspaceAutoReporting(route.params.policyID, isEnabled);
+            },
+            subMenuItems: (
+                <MenuItem
+                    title={translate('workflowsPage.submissionFrequency')}
+                    titleStyle={styles.workspaceWorkflowsSubMenuTitle}
+                    descriptionTextStyle={styles.workspaceWorkflowsSubMenuDescription}
+                    description={translate('workflowsPage.weeklyFrequency')}
+                    shouldShowRightIcon
+                    wrapperStyle={styles.workspaceWorkflowsSubMenuContainer}
+                    hoverAndPressStyle={[styles.mr0, styles.br2]}
+                />
+            ),
+            hasBeenToggled: policy?.harvesting?.enabled ?? false,
+        },
+        {
+            icon: Illustrations.Approval,
+            title: translate('workflowsPage.addApprovalsTitle'),
+            subtitle: translate('workflowsPage.addApprovalsDescription'),
+            onToggle: (isEnabled: boolean) => {
+                Policy.setWorkspaceApprovalMode(route.params.policyID, policy?.owner ?? '', isEnabled ? 'BASIC' : 'OPTIONAL');
+            },
+            subMenuItems: (
+                <MenuItem
+                    title={translate('workflowsPage.approver')}
+                    titleStyle={styles.workspaceWorkflowsSubMenuTitle}
+                    descriptionTextStyle={styles.workspaceWorkflowsSubMenuDescription}
+                    description={policy?.owner ?? ''}
+                    // onPress={() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_APPROVER.getRoute(route.params.policyID))}
```

Comment:
Are we handling this in a follow up?

Reply:
Yes I've added the issues in the comments